extensions/two-factor-auth/package.json

```diff
@@ -0,0 +1,35 @@
+{
+  "name": "two-factor-auth",
+  "title": "Two-Factor Auth",
+  "description": "Two-factor authentication for Raycast",
+  "icon": "command-icon.png",
+  "author": "cjdenio",
+  "license": "MIT",
+  "commands": [
+    {
+      "name": "index",
+      "title": "Two-Factor Auth",
+      "description": "Two-factor authentication for Raycast",
+      "mode": "view"
+    }
+  ],
+  "dependencies": {
+    "@raycast/api": "^1.25.0",
+    "hi-base32": "^0.5.1",
+    "node-fetch": "^2.6.1"
```

Comment:
You might want to remove `node-fetch` dependency since ins't used anywhere 🙂 (also `@types/node-fetch`